tests/unit-tests/webhooks/sequelize/sequelizeConnector-test.js

```diff
@@ -88,7 +97,36 @@ describe('Sequelize client tests', function () {
                 expect(sequelizeGetStub.args[0][0]).to.be.deep.equal({ include: ['events'] });
 
                 expect(webhooks).to.be.an('array').and.have.lengthOf(1);
-                expect(webhooks[0]).to.be.deep.equal(webhookRaw);
+                expect(webhooks[0]).to.be.deep.contain(webhookRaw.dataValues);
+                expect(webhooks[0].events).to.be.deep.equal(webhookRaw.events);
+            });
+        });
+    });
+
+    describe('createWebhook', function() {
+        describe('Happy flow', function() {
+            it('expect to return a webhook with a valid uuid', async function() {
```

Comment:
let's create a test with creating a webhook with `global: true` since it is defaulted to `global: false` and we haven't tested the other case (where it's true)